### DISCUSSION TITLE
Problems installing 3.x.x

### DISCUSSION THREAD

**Discussion by 1537592846** [ColorlibHQ/AdminLTE#5605] [Q&A]
While before it worked without issues, lately, when trying to install admin-lte on version 3.2.0, an error states that I do not have husky installed, which I really don't. This error is preceded by some information of summernote, which we recently stopped using. I've tried creating a new project (Angular 17.3.8) and just adding "admin-lte":"3.2.0" to the package.json, and it did not install, showing the same problem. Any ideas of what it could be? Installing summernote and husky is not inconceivable, but it is not preferable if possible.

## How to recreate the problem

1. Install Angular (npm install -g @angular/cli@17)
2. Create a new project (ng new my-app)
3. Add on the package.json the use of admin-lte ("admin-lte":"3.2.0")
4. Install dependencies (npm install)

**Comment by danny007in** [comment]
is this issue still exist ?

**Reply by mayur2109** [reply]
Yes it still exists
